demo/demo_instructions.rst

```diff
@@ -25,10 +25,21 @@ There are three sites (Jacksonville, UCSD, Omaha) currently available on IARPA's
 
 Valid AWS credentials will need to be passed to the script to access the CORE3D dataset. To do this, use the ``--key_id`` and ``--secret_key`` arguments. It's possible that more sites will be added to the dataset in the future, so to view all available sites, use the ``--show-sites`` option. 
 
+Configuring Danesfield
+----------------------
+
+Two configuration files are needed for Danesfield. The first is required to be named ``input_<region>.ini``, where <region> is replaced by the name of the site. An example of what information it needs can be found at https://github.com/Kitware/Danesfield/blob/master/tools/example_vissat_config.json.
+The only fields that need to be filled out in this file are ``gsd`` and ``cuda``. The first is the desired ground sample distance (default .25), and the second is whether or not CUDA is available. If Danesfield is being run without images, the gsd field can be ignored. 
+The other configuration file is for VisSat Satellite Stereo, and it must be named ``<region>_config.json``. An example of this file can be found at https://github.com/Kitware/Danesfield/blob/master/input.ini. The ``bounding_box`` fields are related to the area that you want to reconstruct with Danesfield. They include the UTM zone, easting, and northing of the bounding box's upper left corner, as well as its width and height in meters. The ``steps_to_run`` fields should be as they appear in the example. The final fields, ``alt_min`` and ``alt_max``, are the approximate minimum and maximum altitudes present in the area of interest. This file is not necessary if Danesfield is being run without images.
```

Comment:
Paragraph about config.json needs a matching reference to https://github.com/Kitware/Danesfield/blob/master/tools/example_vissat_config.json, not to an ini.